crates/tabby/src/services/mod.rs

```diff
@@ -4,3 +4,12 @@ pub mod completion;
 pub mod event;
 pub mod health;
 pub mod model;
+
+pub const DEFAULT_TEMPERATURE: f32 = 0.1;
+
+pub fn default_seed() -> u64 {
```

Comment:
I feel this better being a static method of `TextInferenceEngine`

Reply:
Put it on `TextGenerationOptions`; `TextInferenceEngine` is not accessible from where it's needed for that